projects/overlook.md

```diff
@@ -143,76 +166,118 @@ password: overlook2021
+<section class="answer">
+### UI/UX & Accessibility
 
-## JavaScript Style & OOP
+* The application can stand on its own to be used by an instructor without guidance from a developer on the team.
+UI/UX is intuitive and easy to read/use
+* Helpful messaging is displayed to prevent user confusion. For example: For example: If a user searches for a room, but none are available then a message is displayed to indicate that the search worked, nothing is broken, there just aren't any matching rooms available.
+* The Lighthouse accessibility audit score is at least 90%
+* The application is fully tabbable 
 
-* 4: Application has exceptionally well-factored code with little or no duplication.  The business-logic code driving functionality is cleanly separated from rendering, view-related code.   Excellent usage of `fetch` and updates DOM based on results of network requests.  Handles all scenarios for error handling.
-* 3: Application is thoughtfully put together with some duplication.  Application is organized into classes with some misplaced logic. Business-logic code is mostly separated from view-related code.  Great usage of `fetch` and updates DOM based on results in most scenarios, but may update DOM before a network request is complete.  Handles most scenarios for error handling.
-* 2: Class methods use a mix of array and object prototypes and for loops. Application runs but the code has long methods, unnecessary or poorly named variables, and needs significant refactoring.  Uses `fetch` effectively for `GET` but does not implement `POST`.  Has little error handling and only `console logs` errors if a network request fails.
-* 1:  Application generates syntax error or crashes during execution.  Application is not separated into classes and there is no separation of business-side logic and view-related code. Developer writes code with unnecessary variables, operations, or steps that do not increase clarity.
+✨WOW✨ can look like: 
 
+* Design is responsive across small, medium and large breakpoints
+* Special consideration was made around accessibility
+</section>
 
-## Testing
+<section class="answer">
+### Fundamental JavaScript and Style / Fetch Competency Examples
+
+* Code is divided into logical components each with a clean, single responsibility
+* Array prototype methods are used to iterate instead of for loops
+* All DOM manipulation is held in the scripts.js file. No DOM manipulation occurs outside of this file.
+* Variables and functions are consistently and appropriately named
+* Code leverages JavaScript's truthy/falsey principles
+* Demonstrates efforts towards making functions pure when possible. **Note: Purity is not possible for every function in a FE application. Strive for it only when it makes sense.**
+* Implements GET and POST Fetch requests
+* When utilizing Fetch, the DOM is updated based on the results of that Fetch
+* Most errors are handled and messages are displayed to a user when an error occurs
```

Comment:
Love these three competencies!  These could be good in the group project specs too!